Cargo.toml

```diff
@@ -21,20 +21,23 @@ tokio = { version = "1.2", features = ["net", "rt"] }
 scoped-tls = "1.0.0"
 slab = "0.4.2"
 libc = "0.2.80"
-io-uring = { version = "0.5.0", features = [ "unstable" ] }
-socket2 = { version = "0.4.4", features = [ "all"] }
+io-uring = { version = "0.5.0", features = ["unstable"] }
+socket2 = { version = "0.4.4", features = ["all"] }
 bytes = { version = "1.0", optional = true }
 
 [dev-dependencies]
 tempfile = "3.2.0"
 tokio-test = "0.4.2"
 iai = "0.1.1"
-futures = "0.3.25"
 criterion = "0.4.0"
+tokio = { version = "1.21.2", features = [] }
 
 [package.metadata.docs.rs]
 all-features = true
 
+[profile.bench]
```

Comment:
This is for flamegraph generation.

Reply:
Is this obvious to most people looking at the Cargo.toml? If so, then fine. If not, maybe a line comment?